pkgs/os-specific/linux/xone/default.nix

```
@@ -31,7 +31,7 @@ stdenv.mkDerivation (finalAttrs: {
   meta = with lib; {
     description = "Linux kernel driver for Xbox One and Xbox Series X|S accessories";
     homepage = "https://github.com/medusalix/xone";
-    license = licenses.gpl2;
+    license = licenses.gpl2Only;
```

```suggestion
    license = licenses.gpl2Plus;
```

https://github.com/medusalix/xone/blob/29ec3577e52a50f876440c81267f609575c5161e/bus/bus.c#L1